pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -25,7 +25,7 @@ import (
 	resourceapi "k8s.io/api/resource/v1alpha3"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/kubernetes/pkg/kubelet/cm/dra/state"
+	state "k8s.io/kubernetes/pkg/kubelet/cm/dra/state"
```

Comment:
unnecessary, since the package name is already `state`

Reply:
done